### PR TITLE
Fix: hide `max_vfolder_count` setting according to the api version

### PR DESCRIPTION
**Pull request by agatha197**
follows https://github.com/lablup/backend.ai-webui/pull/2011

## Problem
Updating `max_vfolder_count` in keypair is allowed only up to API 23 version or lower.
But the `max_vfolder_count` setting UI is always shown on the resource policy modification dialog.

## What's changed
- If the core version is greater than 23, hide the `max_vfolder_count` setting UI in the Modify Resource Policy dialog.
- Update schema.

## How to check
1. Go to the Users - Resource Policies page.
2. Check the `Max. # of folder` setting UI is displayed when only the API version is 23 or lower.
3. 23.09 endpoint: http://10.82.230.94:8090

## Checklist
- [ ] Main branch(greater than 23): `Max. # of folder` UI disappears on the create and update resource policy dialog.
- [ ] 23 or lower: `Max. # of folder` UI appears.

**Checklist:** (if applicable)

- [x] Mention to the original issue
- [ ] Documentation
- [x] Minium required manager version
- [x] Specific setting for review (eg., KB link, endpoint or how to setup)
- [x] Minimum requirements to check during review
- [ ] Test case(s) to demonstrate the difference of before/after

### CODE REVIEW

File: react/data/schema.graphql [hunk 1]
```diff
@@ -1,21 +1,37 @@
-# PR #1680 27ab15e5297862f0bf2717b6b0b98d99eb609e26
-
 schema {
```
Comment (yomybaby) [inline react/data/schema.graphql:3]: Please add the PR number and commit hash to the first line.

Comment (yomybaby) [reply react/data/schema.graphql:3]: Except for this, everything else is LGTM.

Comment (agatha197) [reply react/data/schema.graphql:3]: Done! Thanks :)

File: src/components/backend-ai-credential-view.ts [hunk 2]
```diff
@@ -1475,14 +1475,25 @@ export default class BackendAICredentialView extends BackendAIPage {
...
-              <div class="vertical layout flex">
-                <mwc-textfield label="${_t(
-                  'credential.Max#',
-                )}" class="discrete" id="vfolder-count-limit" type="number" min="0" max="50"
-                    @change="${(e) =>
-                      this._validateResourceInput(e)}"></mwc-textfield>
-              </div>
+            <div
+              class="horizontal layout justified"
+              style=${
```
Comment (lizable) [inline src/components/backend-ai-credential-view.ts:1489]: During rendering period, global variable (globalThis.backendaiclient.###) is set to undefined, which triggers runtime errors.  Please add additional property or variable to check whether manager version supports `max-vfolder-count-in-user-resource-policy`.
You may refer similar cases in here:
- [`container-registry-gql` support checking property in backend-ai-environment-view](https://github.com/lablup/backend.ai-webui/blob/51078d876a568db640dd6d5af2c760304de2fbed/src/components/backend-ai-environment-view.ts#L115-L116C31)
- [`inference-workload`, `sftp-scaling-group` support checking property in backend-ai-sessiojn-view](https://github.com/lablup/backend.ai-webui/blob/51078d876a568db640dd6d5af2c760304de2fbed/src/components/backend-ai-session-view.ts#L187-L190)

**Review by lizable — CHANGES_REQUESTED** [review]
I got runtime error when testing on upstream version (`24.03.0dev0`) of Backend.AI Core. Please refer to the comments below and fix it. I think it's easy to handle.

File: src/components/backend-ai-credential-view.ts (hunk 2 above)
Comment (agatha197) [reply src/components/backend-ai-credential-view.ts:1489]: Good point! Thank you. I reflected it.

**Review by lizable — APPROVED** [review]
LGTM.

**Pull request merged**